source/includes/steps-install-verify-files-windows.yaml

```diff
@@ -0,0 +1,129 @@
+title: Download the Signature Check utility from Microsoft.
```

Comment:
Being that this appears to be called Sigcheck, I think we should either say "Download the Sigcheck utility..." or lowercase "signature check"

Reply:
Changed to SigCheck.

Reply:
Nit: According to the linked page the 'c' is lowercase, "Sigcheck"